### ISSUE TITLE
Dev branch add vendor/autoload and compose to dockerfile

### ISSUE THREAD

**Issue by jaycrossler** [akmolina28/last-watch-ai#28]
I'm trying to use the latest changes you added (yay mqtt!) but running into some problems with the docker build.  I think there's a missing ../vendor/ directory, likely because composer wasn't run in the docker install.  I'm not sure what should be in that to finish testing.

**Comment by akmolina28** [comment]
Yes, there is a new composer dependency for the mqtt client so that needs to be installed, along with some database and vue changes. After pulling the latest from master you need to install the missing dependencies, run the migrations and re-compile the vue code, etc. I have a command that does all of the rebuild steps that I run after every upgrade to make it simple:

```
sudo docker-compose run --rm composer install --optimize-autoloader --no-dev &&
sudo docker-compose run --rm artisan route:cache &&
sudo docker-compose run --rm artisan migrate --force &&
sudo docker-compose run --rm npm install --verbose &&
sudo docker-compose run --rm npm rebuild &&
sudo docker-compose run --rm npm run prod --verbose
```

See here for full upgrade instructions: https://kleypot.com/last-watch-ai-ubuntu-installation-and-upgrading/#upgrading-from-source

**Comment by jaycrossler** [comment]
Thanks, I made a windows version (I keep the .env in the parent directory):

```
cd last-watch-ai
docker-compose down
git pull
copy ..\.env .
docker-compose up -d mysql
docker-compose run --rm composer install --optimize-autoloader --no-dev
docker-compose run --rm artisan route:cache
docker-compose run --rm artisan migrate --force
docker-compose run --rm npm install --verbose
docker-compose run --rm npm rebuild
docker-compose run --rm npm run prod --verbose
docker-compose up -d --build site
```

But I'm getting a 500 error on the :8080 site.  Any suggestions where to look for the logs?

**Comment by akmolina28** [comment]
My bad I assumed you were on Linux. Not sure why you see that but usually that happens when you try to migrate before the database is ready. I see that happen sometimes if I run the migrations in a script right after starting the container.

I did test just now on Windows Server 2019 and everything is running fine. I added a new release too if you would rather just use the zipped version that I built.

https://github.com/akmolina28/last-watch-ai/releases/tag/0.8.0-alpha
